javascriptv3/example_code/cross-services/ddb-item-tracker/README.md

```diff
@@ -0,0 +1,729 @@
++  This code has not been tested in all AWS Regions. Some AWS services are available only in specific regions. For more information, see [AWS Regional Services](https://aws.amazon.com/about-aws/global-infrastructure/regional-product-services).
++ Running this code might result in charges to your AWS account.
++ Be sure to terminate the resources you create while going through this tutorial to ensure that you’re not charged.
+
+
+### <a name="createTheResources"></a>Create the resources
+You need the following resources for this app:
+1. An Amazon Cognito unauthenticated user role
+2. Amazon IAM permissions attached to the user role
+3. An Amazon DynamoDB table named 'Work', with a primary key named 'id', of type 'STRING'
+4. An Amazon S3 bucket (this is used to store the CSV file)
+5. Two email addresses verified in Amazon SES
+
+
+You can create the AWS resources required for this cross-service example using either of the following:
+- [Option 1: Amazon CloudFormation](#create-the-resources-using-amazon-cloudformation)
+- [Option 2: The AWS Management Console](#create-the-resources-using-the-aws-management-console)
+
+#### <a name="create-the-resources-using-amazon-cloudformation"></a>Option 1: Create the resources using Amazon CloudFormation
+
+You can run a single Amazon CloudFormation script to provision all the required resources for this example.
+
+1. Install and configure the AWS CLI following the instructions in the AWS CLI User Guide.
+
+2. Open the AWS Command Console from the *./ddb-item-tracker* folder.
+
+3. Run the following command, replacing *STACK_NAME* with a unique name for the stack.
+```
+aws cloudformation create-stack --stack-name STACK_NAME --template-body file://setup.yaml --capabilities CAPABILITY_IAM
+```
+**Important**: The stack name must be unique within an AWS Region and AWS account. You can specify up to 128 characters, and numbers and hyphens are allowed.
+
+4. Open [AWS CloudFormation in the AWS Management Console](https://aws.amazon.com/cloudformation/), and open the **Stacks** page, and chose the **Resources** tab.
+
+![ ](images/cloud_formation_stacks.png)
+
+5. Choose the **Resources** tab. The resources required for this cross-service example are listed in the **Physical ID** column.
+
+![ ](images/cloud_formation_resources_tab.png)
+
+For more information on the create-stack command parameters, see the [AWS CLI Command Reference guide](https://docs.aws.amazon.com/cli/latest/reference/cloudformation/create-stack.html), and the [AWS CloudFormation User Guide](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/using-cfn-cli-creating-stack.html).
+
+#### <a name="create-the-resources-using-the-aws-management-console"></a>Option 2: Create the resources using the AWS Management Console
+
+#### 1. Create an unauthenticated user role
+1. Open [AWS Cognito in the AWS Management Console](https://aws.amazon.com/cloudformation/), and open the *Stacks* page.
```

Comment:
This links to CloudFormation and you refer to "Stacks", but I think you mean Cognito link (and something other than Stacks).

Reply:
done